earth_enterprise/src/server/wsgi/wms/ogc/service/wms_request_app.py

```diff
@@ -57,6 +57,11 @@ def __call__(self, environ, start_response):
       response_headers.append(content_type)
 
     response_body = output
+    try:
+      response_body = response_body.encode('ascii')
+    except (UnicodeDecodeError, AttributeError):
+      pass
```

Comment:
Shouldn't we at least log something?

Reply:
This isnt an error so to speak. Essentially, we previously were returning images as strings, and now we return them as bytes, which we dont want to encode. This allows us to handle both scenarios. Logging here would fill the logs. I could reformat this as if statement, if that would be easier to read.

Reply:
I think the best solution here is <drumroll please> ... a code comment explaining what you just said there :)

Reply:
Will do.